source/release-notes/1.8.txt

```diff
@@ -21,9 +21,91 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.8, ``mongosync`` includes an optional embedded
+verifier that performs a series of checks against the source and 
+destination clusters to verify a successful sync.
+
+For more information, see :ref:`c2c-embedded-verifier`.
+
+Other Notes
+~~~~~~~~~~~
+
+Updates to ``mongosync`` ``/progress`` API endpoint:
```

Comment:
(question) 
This _looks_ to be a big update. Is it worth making this its own section? Also, are there doc updates that you could link out to?

Reply:
I think moving to it's own section makes sense. I have moved it and added a link to the `/progress` documentation page.